doc/api/v0.yaml

```diff
@@ -175,3 +198,22 @@ components:
         fund_id:
           type: integer
           format: int32
+
+    Challenge:
+      properties:
+        title:
+          type: string
+        description:
+          type: string
+        extended_description:
+          type: string
+        rewards_total:
+          type: integer
+          format: int32
```

Comment:
If this is in lovelace, should probably be int64 just to be sure?

Reply:
Yeah, sounds right.